cfep-20.md

```diff
@@ -0,0 +1,97 @@
+
+<table>
+<tr><td> Title </td><td> Standard package split (dev / doc / dbg) </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Sylvain Corlay &lt;sylvain.corlay@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Oct 21, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy on how to split packages in multiple outputs for
```

Comment:
We may want to specify exactly which kinds of packages we are referring to here.

Reply:
I think this should be for all packages.

Reply:
So we won't ship headers with numpy in your view?

Reply:
I responded to that in the thread below:

numpy headers are not installed in a header directory (they are package data). All package data should probably be installed alongside the runtime package.

Reply:
Yup! I missed that when I sent this message or got my own wires crossed. Your answer makes sense. Thank you!